labelbox/schema/identifiable.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Union
+
+
+class IdType(str, Enum):
+    DataRowId = "UID"
+    GlobalKey = "GLOBAL_KEY"
+
+
+class Identifiable(ABC):
+
+    def __init__(self, keys: Union[str, List[str]], id_type: IdType):
+        self._keys = keys
+        if isinstance(keys, str):
+            self._keys = [keys]
+        self._id_type = id_type
+
+    @property
+    def keys(self):
+        return self._keys
+
+    @keys.setter
+    def keys(self, keys):
+        self._keys = keys
+        if isinstance(keys, str):
+            self._keys = [keys]
+
+    @classmethod
+    @abstractmethod
+    def strings_to_identifiable(cls, keys: Union[str, List[str]]):
```

Comment:
Here there is no reason to duplicate the argument type being passed in.  (strings_). The function params already says its strings.  Many libs use like of() or from(), like:

```
keys  = GlobalKeys.of(["some key"])
```

For example in spanner library, its ```Key.of("value")```